### PR TITLE
US#27147 CP3 API Documentation for CPD Study Programs

### PR DESCRIPTION
**Pull request by mkhan-AG**
### Description of changes
* Added documentation for completed and not deprecated study programs

### Type
- Bug fix

### Testing

Readme:
image

Study Programs Readme:
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,11 @@ The CentrePoint API uses [OAuth 2.0](https://oauth.net/2/) for its authorization
 * Retrieval of listing of webhook events. (see [Webhook Events](./sections/webhook_events.md))
 * Retrieval of a list of the webhook subscriptions for a particular study as well as the history or webhook requests. (see [Webhooks](./sections/webhooks.md))
 
+#### CentrePoint Designer
+
+* Retrieval of completed and not depreciated Study Programs. (see [Study Programs](./sections/study_program.md))
```
Comment (mark-fogle) [inline README.md:37]: should be "deprecated"

File: sections/study_programs.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
+# CentrePoing Designer Study Programs
```
Comment (mark-fogle) [inline sections/study_programs.md:1]: (sp?) CentrePoint

File: sections/study_programs.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+**NOTE: These requests require the *CentrePoint* API scope. (see [Scopes](scopes.md))**
+
+## List of Study Programs
+
+Returns a list of study programs that are completed and not depreciated within your organization.
```
Comment (mark-fogle) [inline sections/study_programs.md:7]: deprecated

File: sections/study_programs.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+{
+    "studyProgramId": "1f08d07a-a507-47df-bc89-09412ae42352",
+    "studyId": 234,
+    "studyProgramName": "LeapBarometer (17)"
+}
+```
+
+## Get Study Program Dataset Names
+
+Returns a list of study program dataset names for study programs that are completed and not depreciated.
```
Comment (mark-fogle) [inline sections/study_programs.md:104]: deprecated.

**Review by mark-fogle — CHANGES_REQUESTED** [review]
Some spelling errors. Since this is a public API and it will always return completed and not deprecated, and since those are internal terms, we might not even need to mention that.

**Review by wscott3 — CHANGES_REQUESTED** [review]
I agree with Mark. All resources are simply available or not through this API. Please remove all "deprecated" and "completed". You can use "available" if need.

File: README.md (hunk 1 above)
Comment (mkhan-AG) [reply README.md:37]: I had removed that section for CentrePoint Designer. Moved the reference line referring to the Study Programs documentation.

File: sections/study_programs.md (hunk 2 above)
Comment (mkhan-AG) [reply sections/study_programs.md]: Based on review with Amulya I had removed the heading for CentrePoint Designer Study Programs. Moved the reference to Study Programs documentation to be under Export Dataset Files heading.

File: sections/study_programs.md (hunk 3 above)
Comment (mkhan-AG) [reply sections/study_programs.md]: Yes, I have corrected all the typos pertaining to deprecated.

The PR is now ready for your review!

File: sections/study_programs.md (hunk 4 above)
Comment (mkhan-AG) [reply sections/study_programs.md]: Same reply as before.

**Review by mark-fogle — APPROVED** [review]

**Pull request merged**